src/inflation/model/price.clj

```diff
@@ -0,0 +1,8 @@
+(ns inflation.model.price)
+
+(defn new-price [country type value year month]
```

Comment:
Avoid functions that have several positional parameters, they are hard to extend and make the client code difficult to read. In this case the client code can just create the map (and you don't need this function).

Reply:
I understand your position, but for me still when I use a map as a parameter, I am forcing the client to know how to build the map and this is a dilemma for me. 
How can I make the client know how he should create the map? @rfhayashi

Reply:
~ thoughts java 🤣

Reply:
It depends a lot on the context. In your case, the datomic schema tells that (https://github.com/k-batista/inflation/pull/1/files#diff-9cf35d4eb880e1cb7b2a7ce77a7645311fa5e362ad18d3ddff1dafd7c36c0363R13). Specs/schemas can help as well (https://clojure.org/guides/spec / https://github.com/plumatic/schema), although I'm not a super fan to use them in function level.